course_content/notebooks/numpy_intro.ipynb

```diff
@@ -862,11 +864,10 @@
      "metadata": {},
      "source": [
       "#### Broadcasting: Summary of key  points\n",
+      "\n",
```

Comment:
I think we need to use newlines to separate lists from other markdown elements (headers, paragraphs etc.). Otherwise I don't think they render properly in nbviewer.